### PR TITLE
feat(822): add pr to event

### PR DESCRIPTION
**Pull request by lusol**
add pr url to returned event object

related pr: https://github.com/screwdriver-cd/screwdriver/issues/822

### CODE REVIEW

**Review by tkyi — CHANGES_REQUESTED** [review]
Can you add/modify a test?

**Review by minzcmu — CHANGES_REQUESTED** [review]
Commit message should start with `fix` or `feat`. And tests?

**Comment by lusol** [comment]
added a small test

File: lib/eventFactory.js [hunk 1]
```diff
@@ -291,6 +292,10 @@ class EventFactory extends BaseFactory {
                 return p.sync();
             })
             .then((p) => {
+                if (config.prInfo) {
```
Comment (pranavrc) [inline lib/eventFactory.js]: `if (config.prInfo && config.prInfo.url)  {` perhaps?

Comment (lusol) [reply lib/eventFactory.js]: ok

**Review by tkyi — APPROVED** [review]
LGTM 👍

**Review by minzcmu — APPROVED** [review]
👍

**Pull request merged**